jobs/build_debian/build_debian.sh

```diff
@@ -2,9 +2,12 @@
 set -ex
 pushd $WORKSPACE
 
+LOCAL_BUILD_DIR=b
+ARTIFACTORY_URL=http://afeossand1.cec.lab.emc.com/artifactory
```

Comment:
Agree with @PengTian0 , make ARTIFACTORY_URL as an env var will be more convenient